### ISSUE TITLE
hledger-ui incompatible with latest brick

### ISSUE THREAD

**Issue by snoyberg** [simonmichael/hledger#1882]
Discovered on Stackage build:

```
/var/stackage/work/unpack-dir/unpacked/hledger-ui-1.26-1fbb857f2f188749790deec4f21a7a2b34e787b261217b566e39b6cadbc4243e/Hledger/UI/RegisterScreen.hs:305:65: error:
    • Couldn't match expected type ‘BrickEvent Name e0’
                  with actual type ‘Event’
    • In the first argument of ‘handleEditorEvent’, namely ‘ev’
      In a stmt of a 'do' block: ed' <- handleEditorEvent ev ed
      In the expression:
        do ed' <- handleEditorEvent ev ed
           continue $ ui {aMode = Minibuffer "filter" ed'}
    |
305 |         VtyEvent ev              -> do ed' <- handleEditorEvent ev ed
    |                                                                 ^^
```

**Comment by snoyberg** [comment]
Same concept applies to hledger-iadd:

```
/var/stackage/work/unpack-dir/unpacked/hledger-iadd-1.3.17-4aba586741db8c373a9674dee6167274a0bb0aa7c79c7332d3b7e0beda403942/src/Brick/Widgets/Edit/EmacsBindings.hs:86:36: error:
    • Couldn't match expected type ‘BrickEvent n e0’
                  with actual type ‘Event’
    • In the first argument of ‘E.handleEditorEvent’, namely ‘event’
      In a stmt of a 'do' block:
        newOrig <- E.handleEditorEvent event (edit ^. origEditor)
      In the expression:
        do newOrig <- E.handleEditorEvent event (edit ^. origEditor)
           return $ edit & origEditor .~ newOrig
    • Relevant bindings include
        edit :: Editor n
          (bound at src/Brick/Widgets/Edit/EmacsBindings.hs:71:25)
        handleEditorEvent :: Event -> Editor n -> EventM n (Editor n)
          (bound at src/Brick/Widgets/Edit/EmacsBindings.hs:71:1)
   |
86 |     newOrig <- E.handleEditorEvent event (edit^.origEditor)
   |                                    ^^^^^
```

**Comment by snoyberg** [comment]
CC @hpdeifel regarding hledger-iadd ^^

**Comment by simonmichael** [comment]
Thanks @snoyberg. I have released hledger-ui-1.26.1 with a fix.
